doc/en/user/source/extensions/csw-iso/queryables.rst

```diff
@@ -0,0 +1,14 @@
+CSW ISO Metadata Profile Queryables
+===================================
+
+Mapping
+~~~~~~~
+The ISO Metadata standard (see `OGC Implementation Specification 07-045 <http://www.opengeospatial.org/standards/specifications/catalog>`_) specifies a list of so called 'queryables'. These are simple property names such as 'Title' or 'Abstract' that can be used in filters and will automatically be translated to a corresponding XPath.
```

Comment:
I remember things slightly differently, "queriables" are a construct of CSW that advertises a uniform name for the queryable regardless of the underlying language being used (e.g., 'Title' should work with dublin core as well).

This is from the CSW 2.0 core spec:

![image](https://github.com/geoserver/geoserver/assets/325433/beb24c57-7167-4e6d-8477-b6cad7f9e431)

And quoting the text:

> The goal of defining core queryable properties is query interoperability among catalogues
that implement the same protocol binding and query compatibility among catalogues that
implement different protocol bindings perhaps through the use of “bridges” or protocol
adapters. Defining a set of core queryable properties also enables simple cross-profile
discovery, where the same queries can be executed against any catalogue service without
modification and without detailed knowledge of the catalogue's information model

Long story short, a queryable mapping mechanism should be general.

Reply:
You are right, I didn't even know. It is a separate issue though, so I created one on jira: https://osgeo-org.atlassian.net/browse/GEOS-11323

Reply:
I have changed the docs a bit too